phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/MainWindowView.java

```diff
@@ -71,7 +74,7 @@ public void setCenter(Node nodeToShow) {
         this.content.setCenter(nodeToShow);
     }
 
-    public void showDetailsView(Node nodeToShow) {
+    public void showDetailsView(DetailsView nodeToShow) {
```

Comment:
This change forces us to provide details views with a close button. I'm not sure if we want to allow details views that can't be closed by the user.